src/axom/core/ArrayView.hpp

```diff
@@ -86,6 +86,17 @@ class ArrayView : public ArrayBase<T, DIM, ArrayView<T, DIM, SPACE>>
                              const StackArray<IndexType, DIM>& shape,
                              IndexType spacing_ = 1);
```

Comment:
stride better than spacing? Why the underscore?

Reply:
Maybe replace `spacing_` with `minStride`.  Note that specifying minimum stride is actually incomplete, and this constructor also assumes row-major layout, the only layout we support up to this point.  With this PR, we can now support column-major, arbitrary layouts (though that's not a feature apps are clamoring for) and blueprint stride specifications.